### ISSUE TITLE
Lack of data in window causes delays and errors

### ISSUE THREAD

**Issue by debonair99** [timelyart/Kairos#29]
When there is a lack of data available in the window, there is a delay and errors
![Clipboard - October 26, 2021 2_40 PM](https://user-images.githubusercontent.com/85971842/138990156-b821490d-bb4c-4cd7-8630-8d554427fb4c.png)

**Comment by timelyart** [comment]
At first glance, I cannot reproduce this.

```
2021-10-27 11:00:27.594 INFO tv - process_symbol: NASDAQ:VTYX
2021-10-27 11:00:27.603 INFO tv - element_exists: True (//div[not(contains(@class, "hidden"))]/div[@class="chart-data-window-header"]/span[starts-with(text(), "TRI MAs")][1])
2021-10-27 11:00:28.096 INFO tv - element_exists: True (div.widgetbar-page.active > div.widgetbar-widget.widgetbar-widget-datawindow)
```

Since, I don't see any data under TRI MAs at all and the SYMBOL box appears to have focus did you by chance use the computer while running Kairos? Whether, you did or not - I suggest running Kairos in background mode on Windows machines so Kairos doesn't interfere with your use of the machine (and vice versa).

If that doesn't help, can you post your **[kairos.cfg](https://github.com/timelyart/Kairos/blob/master/_kairos.cfg)** as well as the **YAML file(s)**? *(without any data sensitive information)*

**Comment by debonair99** [comment]
Have tried running Kairos in the background and the issue persists for all tickers that don't have data in the window. This would be due to a lack of volume that was traded over a period of time as all these tickers are relatively newly listed. I can overide manually using the spacebar but if leaving Kairos to run it delays its overall operation
![image](https://user-images.githubusercontent.com/85971842/139158985-f3ea4e59-243c-4731-bb1e-d6f86918de3c.png)
![image](https://user-images.githubusercontent.com/85971842/139159019-0ea04215-8bde-476f-8d9f-90c9cebc8aad.png)
Yaml in text->
[yaml in text.txt](https://github.com/timelyart/Kairos/files/7429828/yaml.in.text.txt)
CFG in text->
[CFG in text.txt](https://github.com/timelyart/Kairos/files/7429835/CFG.in.text.txt)

**Comment by timelyart** [comment]
Thanks for providing the requested files. With your YAML file I was able to reproduce the issue.
Fixed with [release v2.54](https://github.com/timelyart/Kairos/releases/tag/v2.54).
